components/gucharmap/gucharmap.p5m

```diff
@@ -242,3 +242,12 @@ file path=usr/share/omf/gucharmap/gucharmap-vi.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_CN.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_HK.omf
 file path=usr/share/omf/gucharmap/gucharmap-zh_TW.omf
+
```

Comment:
please, use gmake sample-manifest and add necessary actions in corresponding place